_posts/JayYang95/2016-02-01-ClickyTurtle.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: JayYang95
+title: "Jay's Clicky Turtle Exercise"
+---
+
+<iframe src="https://trinket.io/embed/python/ff73c1dfa7" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Nice!  I like it.

Check out `random.choice()`- it's an easier way to do what you did with selecting a random item from a list (which was very creative).  We'll cover that when we get to lists.